exporter/collector/metricsexporter.go

```diff
@@ -227,8 +229,10 @@ func (m *metricMapper) resourceMetricsToMonitoredResource(
 }
 
 func instrumentationLibraryToLabels(il pdata.InstrumentationLibrary) labels {
-	// TODO
-	return nil
+	return labels{
+		"instrumentation_source":  il.Name(),
```

Comment:
Should this add source or version if the name is empty?   cc @kjordy / @aabmass

Reply:
Discussed. We will set to empty for now, since it is currently unspecified.